### ISSUE TITLE
Missing function `:crypto.block_encrypt/4 ` in Erlang 24

### ISSUE THREAD

**Issue by coladarci** [danielberkompas/cloak_ecto#23]
Just attempted a bump to elixir 1.12 + Erlang 24 and getting a bunch of these:

```
     apps/web/test/web/my_test.exs:22
     ** (UndefinedFunctionError) function :crypto.block_encrypt/4 is undefined or private, use crypto:crypto_one_time/5, crypto:crypto_one_time_aead/6,7 or crypto:crypto_(dyn_iv)?_init + crypto:crypto_(dyn_iv)?_update + crypto:crypto_final instead
     stacktrace:
       (crypto 5.0.1) :crypto.block_encrypt(:aes_gcm, <<195, 225, 165, 141, 41, 78, 62, 205, 98, 180, 191, 110, 234, 172, 72, 184, 62, 87, 136, 164, 190, 51, 242, 243, 163, 183, 119, 33, 180, 102, 137, 119>>, <<120, 219, 37, 247, 92, 246, 21, 28, 46, 15, 17, 201, 227, 63, 247, 245>>, {"AES256GCM", "secret"})
       (cloak 1.0.3) lib/cloak/ciphers/aes_gcm.ex:46: Cloak.Ciphers.AES.GCM.encrypt/2
       (web 0.1.0) lib/cloak_ecto/type.ex:37: Web.Encrypted.Binary.dump/1
       (ecto 3.6.1) lib/ecto/type.ex:914: Ecto.Type.process_dumpers/3
       (ecto 3.6.1) lib/ecto/repo/schema.ex:953: Ecto.Repo.Schema.dump_field!/6
       (ecto 3.6.1) lib/ecto/repo/schema.ex:966: anonymous fn/6 in Ecto.Repo.Schema.dump_fields!/5
       (stdlib 3.15) maps.erl:410: :maps.fold_1/3
       (ecto 3.6.1) lib/ecto/repo/schema.ex:964: Ecto.Repo.Schema.dump_fields!/5
       (ecto 3.6.1) lib/ecto/repo/schema.ex:897: Ecto.Repo.Schema.dump_changes!/6
       (ecto 3.6.1) lib/ecto/repo/schema.ex:335: anonymous fn/15 in Ecto.Repo.Schema.do_insert/4
       (ecto 3.6.1) lib/ecto/repo/schema.ex:942: anonymous fn/3 in Ecto.Repo.Schema.wrap_in_transaction/6
       (ecto_sql 3.6.1) lib/ecto/adapters/sql.ex:1005: anonymous fn/3 in Ecto.Adapters.SQL.checkout_or_transaction/4
       (db_connection 2.4.0) lib/db_connection.ex:1512: DBConnection.run_transaction/4
```

Thanks for the great project!

**Comment by acco** [comment]
I believe this is being worked on here: https://github.com/danielberkompas/cloak/issues/102

**Comment by coladarci** [comment]
Thank you, I literally just found that and was going to add a comment to the affect..

**Comment by danielberkompas** [comment]
This is fixed in version `1.2.0` of `cloak_ecto`, just released.
